src/app/dashboard/freelancer/page.tsx

```diff
@@ -130,64 +162,44 @@ export default function Dashboard() {
 
               <StatCard
                 title="Active Projects"
-                value={
-                  loading
-                    ? '...'
-                    : projects.filter((p) => p.status === 'Active').length
-                }
+                value={loadingStats ? '...' : activeProjects.length}
                 icon={<CheckCircle className="h-6 w-6 text-success" />}
                 additionalInfo={
-                  loading ? 'Loading...' : 'Earning stats will be here'
+                  loadingStats ? 'Loading...' : 'Earning stats will be here'
                 }
               />
               <StatCard
                 title="Pending Projects"
-                value={
-                  loading
-                    ? '...'
-                    : projects.filter((p) => p.status === 'Pending').length
-                }
+                value={loadingStats ? '...' : pendingProjects.length}
                 icon={<Clock className="h-6 w-6 text-warning" />}
                 additionalInfo={
-                  loading ? 'Loading...' : 'Project stats will be here'
+                  loadingStats ? 'Loading...' : 'Project stats will be here'
                 }
               />
             </div>
 
             {/* Tabs for project filtering */}
             <div className="overflow-x-auto">
-              <Tabs defaultValue="active">
+              <Tabs value={currentTab} onValueChange={handleTabChange}>
                 <div className="flex items-center">
                   <TabsList>
-                    <TabsTrigger value="active">Active</TabsTrigger>
-                    <TabsTrigger value="pending">Pending</TabsTrigger>
-                    <TabsTrigger value="completed">Completed</TabsTrigger>
-                    <TabsTrigger value="rejected">Rejected</TabsTrigger>
+                    <TabsTrigger value="Active">Active</TabsTrigger>
```

Comment:
I think we have enums for these states Use from there. If not then create enum and use from there.